tests/test_hawk_http_backend.py

```diff
@@ -0,0 +1,45 @@
+from cachish.backends import Hawk
+
+import responses
+
+@responses.activate
+def test_hawk_backend():
+    response = {
+        "key1": "value1",
+        "key2": "value2",
+    }
+    responses.add(responses.GET, 'https://example.com',
+        json=response)
+    config = {
+        'secret_id': 'supersecret',
+        'secret': 'youreawizardharry',
+        'algorithm': 'sha256',
+        'url': 'https://example.com',
+    }
+    backend = Hawk(**config)
+    value = backend.get()
+    assert len(responses.calls) == 1
+    assert value == response
+    sent_user_agent = responses.calls[0].request.headers.get('authorization')
```

Comment:
This is not a user agent, variable should be renamed to reflect what it is (auth header).